.github/workflows/integration-test.yaml

```diff
@@ -10,16 +10,17 @@ jobs:
     env:
       GAFFER_VERSION: develop
       GAFFER_TOOLS_VERSION: develop
+      ACCUMULO_VERSION: develop
     steps:    
     - uses: actions/checkout@v2
 
-      # See issue #153 for why Gaffer image is not using docker-compose
     - name: Build images
       run: |
-        docker-compose --project-directory ./docker/gaffer-ui/ -f ./docker/gaffer-ui/docker-compose.yaml build --build-arg GAFFER_TOOLS_VERSION=$GAFFER_TOOLS_VERSION
-        docker build -t gchq/gaffer:$GAFFER_VERSION --build-arg GAFFER_VERSION=$GAFFER_VERSION ./docker/gaffer
+        docker build -t gchq/accumulo:$ACCUMULO_VERSION ./docker/accumulo
```

Comment:
For now this is necessary as Accumulo is not built locally within the Gaffer Dockerfile.